### PR TITLE
[2.6] Improvements to new navigation UI

### PR DESCRIPTION
**Pull request by nwmac**
This PR replaces #2600 

Improves colors, adds dark mode support for slide-in menu.

### CODE REVIEW

File: components/nav/Header.vue [hunk 1]
```diff
@@ -48,9 +48,17 @@ export default {
     },
 
     prod() {
-      const name = this.currentProduct.name;
+      const t = this.$store.getters['i18n/t'];
```
Comment (nwmac) [inline components/nav/Header.vue:51]: Revert

File: components/nav/Header.vue [hunk 2]
```diff
@@ -269,8 +300,9 @@ export default {
 
     > .product {
       grid-area: product;
+      align-items: center;
       position: relative;
-      //display: block;
```
Comment (nwmac) [inline components/nav/Header.vue:273]: Remove comment

File: components/nav/Header.vue [hunk 3]
```diff
@@ -332,14 +364,15 @@ export default {
       width: 40px;
     }
 
-    > .cluster {
+    > .spacer {
```
Comment (nwmac) [inline components/nav/Header.vue:367]: Rename to header-spacer

File: components/nav/TopLevelMenu.vue [hunk 4]
```diff
@@ -218,19 +232,27 @@ export default {
...
-                Home
+                {{ t('nav.home') }}
               </div>
             </nuxt-link>
           </div>
           <div class="category">
-            Explore Cluster
+            {{ t('nav.categories.explore') }}
+          </div>
+          <div v-if="showClusterSearch" class="search">
```
Comment (nwmac) [inline components/nav/TopLevelMenu.vue:249]: Don't use a fixed height when search not shown

File: components/nav/Type.vue [hunk 5]
```diff
@@ -8,6 +8,15 @@ export default {
 
   components: { Favorite },
 
+  filters: {
```
Comment (nwmac) [inline components/nav/Type.vue:11]: Pull in to separate PR

**Review by richard-cox — APPROVED** [review]
